cMake/FreeCAD_Helpers/FindFreeType.cmake

```diff
@@ -0,0 +1,13 @@
+macro(FindFreetype)
+#--------------------FreeType-----------------------
+
+    if(FREECAD_USE_FREETYPE)
+        find_package(Freetype)
+        if(NOT FREETYPE_FOUND)
+            message("===============================================================\n"
+                    "FreeType2 not found. Part module will lack of makeWireString().\n"
```

Comment:
`"FreeType2 not found. Part module will lack functionality of makeWireString().\n"`

Reply:
Again, this error is currently [in master ](https://github.com/FreeCAD/FreeCAD/blob/master/CMakeLists.txt#L963). I would like to leave it as-is for this PR.